### PR TITLE
Swagger documentation normalizer revert throw when method is not supp…

### PR DESCRIPTION
**Pull request by soyuka**
…orted

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #1122
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

File: src/Swagger/Serializer/DocumentationNormalizer.php [hunk 1]
```diff
@@ -171,7 +171,6 @@ private function getPathOperation(string $operationName, array $operation, strin
             case 'DELETE':
                 return $this->updateDeleteOperation($pathOperation, $resourceShortName);
             default:
```
Comment (dunglas) [inline src/Swagger/Serializer/DocumentationNormalizer.php]: this line should be reverted and the `return` statement on the last line added back.

File: tests/Swagger/Serializer/DocumentationNormalizerTest.php [hunk 2]
```diff
@@ -1111,6 +1111,8 @@ public function testNoOperations()
      */
     public function testMethodNotSupported()
```
Comment (dunglas) [inline tests/Swagger/Serializer/DocumentationNormalizerTest.php]: We may want to use custom methods (standard ones like PATCH, but also custom ones like BAN or PURGE).

Comment (soyuka) [inline tests/Swagger/Serializer/DocumentationNormalizerTest.php]: Yeah I skipped the test, I'll add one with custom methods.

File: tests/Swagger/Serializer/DocumentationNormalizerTest.php [hunk 3]
```diff
@@ -1151,6 +1153,65 @@ public function testMethodNotSupported()
...
+            $propertyMetadataFactoryProphecy->reveal(),
+            $resourceClassResolverProphecy->reveal(),
+            $operationMethodResolverProphecy->reveal(),
+            $operationPathResolver,
+            $urlGeneratorProphecy->reveal()
+        );
+
+        $expected = [
+            'swagger' => '2.0',
+            'basePath' => null,
```
Comment (soyuka) [inline tests/Swagger/Serializer/DocumentationNormalizerTest.php]: ping @Simperfit is this correct? I think that this custom method behavior isn't correctly supported no?

**Comment by dunglas** [comment]
The patch looks good to me. Can you add the missing test?

**Comment by soyuka** [comment]
I added the missing test already, but the result seems weird.

**Comment by soyuka** [comment]
Okay, tests is fine after second look. Rebased feel free to merge once green.

**Comment by dunglas** [comment]
Thanks @soyuka!

**Pull request merged**